src/Microsoft.MobileBlazorBindings/Elements/GestureExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using Xamarin.Forms;
+
+namespace Microsoft.MobileBlazorBindings.Elements
+{
+    public static class GestureExtensions
```

Comment:
Is it possible to create specific extensions for all the GestureRecognizers with the generator like `OnTapped` for `TapGestureRecognizer` . So it could be used like 

```csharp
new TapGestureRecognizer { NumberOfTapsRequired = 1 }.OnTapped(OnLabelTapped);
```

Reply:
This is a great idea! I'll keep the generic version in case someone wants to do something more esoteric, or some new event/gesture is added.